importify.cabal

```diff
@@ -49,6 +49,8 @@ executable importify
                      , optparse-applicative
                      , haskell-src-exts
                      , haskell-names
+                     , bytestring
+                     , cereal
```

Comment:
`cereal` is a nice option and a good library. I just wonder why exactly you chose it :)
But I think for filer serialization it's better to use some human-readable formatting. Like JSON. So `aeson` library may be better. Another thing why consider `aeson`: `haskell-names` library we're using for name resolution also uses `aeson` for serializing and deserializing symbols. It might be more convenient to use one library and one format for everything if we need to encode/decode those files manually. Also decreases size of dependencies.